samples/app-anonymous-users/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,95 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.16",
+  "version": "1.0.0",
+  "id": "{{Microsoft-App-id}}",
+  "packageName": "com.teams.sample.appAnonymousUsers",
+  "developer": {
+    "name": "appAnonymousUsers",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "icons": {
+    "outline": "outline.png",
+    "color": "color.png"
+  },
+  "name": {
+    "short": "Anonymous User Support",
+    "full": "Anonymous User Support"
+  },
+  "description": {
+    "short": "This sample shows anonymous users support in meeting apps.",
```

Comment:
This sample shows anonymous users support in `Teams` meeting apps.

Reply:
Fixed